code/api/src/modules/user/resolvers.js

```diff
@@ -26,6 +27,16 @@ export async function create(parentValue, { name, email, password }) {
   }
 }
 
+// Update User
```

Comment:
User Update mutation will take arguments of the User ID and styleResult. It will find a user with the corresponding ID and then update that user with the styleResult and save to the db.